app/views/doctors/transcripts/show.haml

```diff
@@ -0,0 +1,13 @@
+%p
+  %b Description:
+  = @transcript.chat_room.urgent_request.description     
+%p
+  %b Patient:
+  = @transcript.chat_room.patient.name 
```

Comment:
`@transcript` is undefined here, this is because the `Doctors::TranscriptsController` is inheriting the `show` action from the `DoctorsController`. Looking at the show action of the doctors controller, we can see that there is no reference to `@transcript = .....`. This is because the `DoctorsController` has nothing to do with transcripts.

The solution would be to define a `show` action in `Doctors::TranscriptsController`, and have it set the instance variable `@transcript = ...` . OR you could do what you've done in the `Patients::TranscriptsController` and have the controller set the transcript on the show action